src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -492,11 +492,13 @@
                 (.writeObject w (.-node-id ^RuleOrderedActivation c) true)
                 (.writeObject w (.-token ^RuleOrderedActivation c))
                 (.writeObject w (.-activation ^RuleOrderedActivation c))
-                (.writeInt w (.-rule-load-order ^RuleOrderedActivation c))))
+                (.writeInt w (.-rule-load-order ^RuleOrderedActivation c))
+                (.writeBoolean w (.-use-token-identity? ^RuleOrderedActivation c))))
```

Comment:
Clojure makes fields with metadata `:unsynchronized-mutable` private by default.
`RuleOrderedActivation.use-token-identity?` is marked with this metadata, so it is private.

I'd expect 2 things here:
1) a reflection warning at compile time for the call `(.-use-token-identity? ^RuleOrderedActivation c)` since that private field isn't exposed so the clj compiler cannot emit an appropriate callsite for it.
2) a runtime exception thrown when this is evaluated due to the field not being found (via reflection due to (1) already happening).

In `clara.rules.memory` you'll see a protocol `IdentityComparable` that is used in order to be able to implement at "setter" on `RuleOrderedActivation` for this private field. 

There is no protocol function implemented that gives access to this field however. It may be reasonable to add another function to the `IdentityComparable` protocol, such as `use-token-identity?` that will return the value of this field for serialization. This protocol is really only an internal impl detail around this optimization, so that should be fine.

Reply:
Example of testing it:
```clj
(require 'clara.test-fressian)
(in-ns 'clara.test-fressian)
(require '[clara.rules.memory :as mem])

(let [act (mem/->RuleOrderedActivation 1 nil nil 1 false)]
  (test-serde act))
```